app/src/main/java/com/example/ncov19traking/ui/countries/CountriesFragment.kt

```diff
@@ -65,6 +75,15 @@ class CountriesFragment : Fragment() {
                 dashboardViewModel.refreshCountryList()
                 return true
             }
+            R.id.appbar_sort -> {
+                if (isSortedByCases) {
+                    isSortedByCases = false
+                    nCoVRecyclerAdapter.changeSort(isSortedByCases)
+                } else {
+                    isSortedByCases = true
+                    nCoVRecyclerAdapter.changeSort(isSortedByCases)
```

Comment:
repeated code, this can be left out of the condition.

Reply:
fixed